Makefile

```diff
@@ -0,0 +1,48 @@
+TOP ?= $(shell git rev-parse --show-toplevel)
+
+export BP_SDK_DIR ?= $(TOP)/sdk
+export BP_RTL_DIR ?= $(TOP)/rtl
+
+JOBS ?= 8
+PROJECT_DIR ?= $(TOP)/proj/arty-parrot
+
+.PHONY: prep prep_lite prep_bsg bleach_all checkout checkout_sdk checkout_rtl
+.PHONY: gen_proj gen_bit clean_proj
+
+checkout_sdk:
```

Comment:
I wouldn't do a recursive checkout of sdk or rtl. The submodules are updated as needed within their respective repos

Reply:
I see that's how it's done in black-parrot-sim, so let's stick with it for now